src/app/components/navbar/NavBar.test.js

```diff
@@ -90,18 +91,18 @@ describe("NavBar", () => {
             });
         });
 
-        describe("when enabled dataset import", () => {
-            it("should display Datasets", () => {
+        describe("when enabled upload interactives", () => {
```

Comment:
Can you please revert back your changes as we would like to keep this test and create new test case for your feature.  enabled dataset import feature flag is still active

Reply:
there was a duplicate with this, so only replaced the duplicate